### ISSUE TITLE
Post Modified Date Not Updated

### ISSUE THREAD

**Issue by mskian** [TryGhost/Ghost#9520]
After updating to `1.21.6` Last Updated Post date Helper not working 

```
{{date updated_at format='YYYY-MM-DD'}}
{{date updated_at format="DD MMMM YYYY"}}
```

### Example:
I publish the Post on 08 Jun 2017, next update the post on 08 September 2017 It will display the Updated post date (`08 September 2017`)
Again I update the post on 21 March 2018 But it has shown the 08 September 2017 Date only 
Now I revert back to Normal date format helper :( `
 {{date format="D MMMM YYYY"}}`

 **PS: Same issue on Schema Markup & Meta Article Modified Tag**

### Technical details:

* Ghost Version:1.21.6

**Comment by AileenCGN** [comment]
Hey @mskian,

I am not able to reproduce your issue.

When I change the `date` helper in my `post.hbs` to this:

`{{date updated_at format="D MMMM YYYY hh:mm:ss"}}` (_I displayed the time as well for testing purposes_)

I can't see any issue. I took an older post, updated it -> worked fine. Then updated it again after a while, and it still worked fine.

Also the meta data gets updated correctly for me.

Can you please check again and let me know, if I missed steps to reproduce it?

**Comment by mskian** [comment]
Hey @AileenCGN 

Leave Helper Step

- Check it from Casper theme
- Open any one of your Old posts Made any changes and Update it
- Open the post "View Source" & Check Schema Markup Modified Date & Meta article Modified date pls

PS: I try from my side same issue 

Waiting for your Next reply

**Comment by AileenCGN** [comment]
I don't know what exactly happened, but I can totally reproduce it now. It appears to happen to all `updated_at` fields (posts, user, tags).

I am not 100% sure, but might it be, that there's an issue with bookshelf's ['hasTimeStamps'](http://bookshelfjs.org/#Model-instance-hasTimestamps) method, which we're using to update `updated_at` and `created_at`. @kirrg001 can you have a look and see, if you can reproduce it?

**Comment by mskian** [comment]
Post Modified Date Not Updated on Schema Markup & MetaProperty `article:modified_time`

**Comment by AileenCGN** [comment]
> Post Modified Date Not Updated on Schema Markup & MetaProperty article:modified_time

This is only a side effect of what I described. The date in schema.org and open graph data is based on the `updated_at` field.

**Comment by kirrg001** [comment]
I can also reproduce. Might be that the last Bookshelf release has caused it, because this property get's set automatically on update (this does not happen in Ghost). This looks critical.

**Comment by kirrg001** [comment]
Probably caused by https://github.com/bookshelf/bookshelf/pull/1583.
There is already a fix in Bookshelf's master, i've asked when the patch goes out.

**Comment by kirrg001** [comment]
Bookshelf just released **0.13.3**, which fixes this issue. Patch release will happen today.

@mskian Thanks for reporting and sorry for trouble!

**Comment by mskian** [comment]
@kirrg001  welcome :-)
